### DISCUSSION TITLE
Game Updates

### DISCUSSION THREAD

**Discussion by maglorsmith93** [wolveix/satisfactory-server#64] [Q&A] [answered]
Hello, I haven't installed yet, so apologies if this is obvious, but was wondering if I could get some clarification on how game updates are handled? is it automatically handled while the container is running or does the container need to be updated when there is a patch? Also does it redownload the whole game or just the patch? trying to figure out bandwidth and other policies.
Cheers

**Comment by wolveix** [comment, marked as answer]
Hey! No problem at all. The game updates whenever the container starts/restarts (the image does not need to be updated to do this). The game itself is ~5GB, and updates are typically just patches.

**Reply by maglorsmith93** [reply]
Thanks so much for your response.
So if I have the container set to restart daily, I would be pretty safe to get the latest update? If the server was running and the container restarts, I assume players get kicked but would be able to reload from last auto save? Or do you have to go through the server admin setup again after an update?

I'm wanting to get this up and running as reliably as possible and will probably have people contributing from around the world so want to know the best way to handle it. Thanks again

**Reply by wolveix** [reply]
Yeah exactly that. Restart the container daily, and it'll update the game during startup (as long as you haven't set `SKIPUPDATE` to `true`). The server will load up the same save on startup, so you won't need any manual intervention!

**Reply by maglorsmith93** [reply]
Absolutely beautiful, love your work. Thanks again
